docs/source/dev/index.rst

```diff
@@ -43,49 +43,141 @@ For other questions regarding OpenFAST, please contact
     - :ref:`code_style`
```

Comment:
Just above this is a link to Mike Sprague.  Suggest changing this to Rafael Mudafort.